packages/seller-client/src/components/BuyerDetails/BuyerDetails.tsx

```diff
@@ -0,0 +1,100 @@
+import React from 'react';
+import { makeStyles } from '@material-ui/core/styles';
+import { Box, Typography, Button } from '@material-ui/core';
+import { Phone, Mail } from '@material-ui/icons';
+import { Address } from 'generated/graphql';
+import Spacer from 'components/Spacer';
+import Asset from './Asset';
+import RatingStars from './RatingStars';
+
+const ASSET_SIZE = 45;
+const BUTTON_SIZE = 40;
+
+const useStyles = makeStyles(({ palette }) => ({
+  button: {
+    display: 'flex',
+    justifyContent: 'center',
+    alignItems: 'center',
+    width: BUTTON_SIZE,
+    height: BUTTON_SIZE,
+    borderRadius: BUTTON_SIZE / 2,
+    backgroundColor: palette.primary.main,
+    color: 'white',
+    minWidth: BUTTON_SIZE, // needed because mui button's have a min width of 64
+  },
+}));
+
+type Props = {
+  firstName: string;
+  lastName: string;
+  email: string;
+  phoneNumber: string;
+  rating: number;
+  address: Omit<Address, '_id' | '__typename' | 'name'>;
+};
+
+const BuyerDetails: React.FC<Props> = ({
+  firstName,
+  lastName,
+  email,
+  phoneNumber,
+  rating,
+  address,
+}) => {
+  const classes = useStyles();
+  const { country, line1, line2, state, townCity, zipCode } = address;
+  return (
+    <Box>
+      <Box display="flex">
+        <Asset
+          size={ASSET_SIZE}
+          initials={`${firstName[0].toLocaleUpperCase()}${lastName[0].toLocaleUpperCase()}`}
+        />
+        <Spacer width={32} />
+        <Box>
+          <Typography variant="h6">
+            Sold to {firstName} {lastName}
+          </Typography>
+          <RatingStars rating={rating} size={16} />
+        </Box>
+      </Box>
+      <Spacer height={16} />
+      <Box>
+        <Typography variant="overline">ADDRESS</Typography>
+        <Typography variant="body2">
+          {line1}
+          <br />
+          {line2 && (
+            <>
+              {line2}
+              <br />
+            </>
+          )}
+          {townCity}
+          <br />
+          {(state || zipCode) && (
+            <>
+              {state && state}
+              {state && zipCode && ', '}
+              {zipCode && zipCode}
```

Comment:
Don't have to check for existence for these
```suggestion
              {state}
              {state && zipCode && ', '}
              {zipCode}
```